src/reviewBody.js

```diff
@@ -0,0 +1,47 @@
+// This function prepares the review data
+function prepareReviewData(newReview) {
+  const date = new Date().toISOString();
+  const body = {
+    data: {
+      comment: 'string',
+      rating: 0,
+      author: 'string',
+      verified: true,
+      movie: 'string or id',
+      createdAt: date,
+      updatedAt: date,
+      createdBy: 'string or id',
+      updatedBy: 'string or id',

```

Comment:
This doesn't look right to me. Is this really working?

My guess is that you've read the API documentation and found that `comment` should be a string, but that doesn't mean that it should have the value `"string"` (that specific piece of text) but that it should have _some_ piece of text.

Because of `...newReview` on the next line, and the fact that `newReview` will contain `movie`, `comment`, `rating` and `author`, those very important fields will have their values overwritten by the real ones. But you still have `'string or id'` (that exact piece of text) as the value for `createdBy` and `updatedBy` and that's not correct. I'm surprised that it even works – I would have expected the CMS to complain about them.